classes/person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :age, :name, :id, :parent_permission
```

Comment:
- [ ] Defining the getter-setter (attr_accessor) functionality on the `id` property implies we want to be able to modify it. But that's not the case, we mostly only want to be able to read (attr_reader) it. Kindly define the `attr_reader` for the `id` instead.

- [ ] **[OPTIONAL]** We don't need `attr_accessor` or `attr_reader` for the `:parent_permission`. We only need it to verify if a person has permission and we can do that already with the `can_use_services?` method.

Reply:
Done.